src/main/java/com/google/sps/model/TasksClient.java

```diff
@@ -56,4 +65,40 @@
    * @throws IOException if an issue occurs with the tasksService
    */
   Task postTask(String parentTaskListId, Task task) throws IOException;
+
+  /**
+   * Get the all tasks in all the user's task lists
+   *
+   * @param tasksClient Either a mock TaskClient or a taskClient with a valid credential
+   * @return List of tasks from all task lists in user's account
+   * @throws IOException if an issue occurs with the tasksService
+   */
+  static List<Task> getAllTasksFromAllTaskLists(TasksClient tasksClient) throws IOException {
+    List<TaskList> taskLists = tasksClient.listTaskLists();
+    List<Task> tasks = new ArrayList<>();
+    for (TaskList taskList : taskLists) {
+      tasks.addAll(tasksClient.listTasks(taskList));
+    }
+    return tasks;
+  }
+
+  /**
+   * Get the tasks in the user's task lists with the given task list IDs
+   *
+   * @param tasksClient Either a mock TaskClient or a taskClient with a valid credential
+   * @param taskListTitles List of task list IDs which tasks should be obtained from
+   * @return List of tasks from specified task lists in user's account
+   * @throws IOException if an issue occurs with the tasksService
+   */
+  static List<Task> getAllTasksFromSpecificTaskLists(
+      TasksClient tasksClient, List<String> taskListIds) throws IOException {
```

Comment:
Same as above regarding `Set`

Reply:
I have changed `taskListIds` to be a `Set<String>`, thanks!

Reply:
Should I change `List<Task>` to `Set<Task>`?

Reply:
I just added a comment to your tests talking about the order. A `Set` would be neat if you don't want to give any guarantee regarding the order. (You could also use [Collection](https://docs.oracle.com/javase/8/docs/api/java/util/Collection.html) to make it more generic)

Reply:
Set is really neat, I looked a little deeper to see if I could change from List<Task> to Set<Task> but it might affect some of the other servlets so I don't think it might be a good idea at this time.

Reply:
`RESOLVED`